modules/apps/transfer/client/cli/tx.go

```diff
@@ -22,6 +22,7 @@ const (
 	flagPacketTimeoutHeight    = "packet-timeout-height"
 	flagPacketTimeoutTimestamp = "packet-timeout-timestamp"
 	flagAbsoluteTimeouts       = "absolute-timeouts"
+	flagMetadata               = "metadata"
```

Comment:
I think this flag needs to be added below, see line 120. The default value can be an empty string